src/applications/financial-status-report/pages/assets/recreationalVehicles/records.js

```diff
@@ -32,9 +33,10 @@ const recreationalVehicleUISchema = {
     recVehicleAmount: {
       'ui:title':
         'What is the estimated value of all of your trailers, campers, and boats?',
+      'ui:webComponentField': VaTextInputField,
       'ui:options': {
         classNames: 'schemaform-currency-input',
-        widgetClassNames: 'input-size-4 vads-u-margin-bottom--3',
+        width: 'md',
```

Comment:
Did we verify that the validation still works on these new VaTextInput fields?

Reply:
I believe so, I didn't see any errors.

Reply:
It throws an error if I leave it blank: <img width="631" alt="Screenshot 2024-04-08 at 2 49 55 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/1631062/60073f6e-0edf-48b8-b749-d79bf9a5982c">

Reply:
They work!